prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics-release.yaml

```diff
@@ -0,0 +1,1320 @@
+
+- <<: *config-sync-ci-job
+  name: release-multi-repo-1-autopilot-stable
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-1
+    testgrid-tab-name: autopilot-stable
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-gke-multi-repo-test-group1
+      - 'GCP_CLUSTER=multi-repo-1-autopilot-stable'
+      - 'GCP_REGION=us-central1'
+      - 'GCP_ZONE=""'
+
+- <<: *config-sync-ci-job
+  name: release-multi-repo-1-autopilot-regular
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-1
+    testgrid-tab-name: autopilot-regular
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-gke-multi-repo-test-group1
+      - 'GCP_CLUSTER=multi-repo-1-autopilot-regular'
+      - 'GCP_REGION=us-central1'
+      - 'GCP_ZONE=""'
+
+- <<: *config-sync-ci-job
+  name: release-multi-repo-1-autopilot-rapid
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-1
+    testgrid-tab-name: autopilot-rapid
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-gke-multi-repo-test-group1
+      - 'GCP_CLUSTER=multi-repo-1-autopilot-rapid'
+      - 'GCP_REGION=us-central1'
+      - 'GCP_ZONE=""'
+
+- <<: *config-sync-ci-job
+  name: release-multi-repo-1-autopilot-rapid-latest
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-1
+    testgrid-tab-name: autopilot-rapid-latest
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-gke-multi-repo-test-group1
+      - 'GCP_CLUSTER=multi-repo-1-autopilot-rapid-latest'
+      - 'GCP_REGION=us-central1'
+      - 'GCP_ZONE=""'
+
+- <<: *config-sync-ci-job
+  name: release-multi-repo-1-standard-regular-bitbucket
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-1
+    testgrid-tab-name: standard-regular-bitbucket
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-gke-multi-repo-test-group1
+      - 'E2E_ARGS=--git-provider=bitbucket'
+      - 'GCP_CLUSTER=multi-repo-1-standard-regular-bitbucket'
+
+- <<: *config-sync-ci-job
+  name: release-multi-repo-1-standard-regular-gitlab
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-release-multi-repo-1
+    testgrid-tab-name: standard-regular-gitlab
+  spec:
+    <<: *config-sync-ci-job-spec
+    containers:
+    - <<: *config-sync-ci-container
+      args:
+      - make
+      - test-e2e-gke-multi-repo-test-group1
+      - 'E2E_ARGS=--git-provider=gitlab'
+      - 'GCP_CLUSTER=multi-repo-1-standard-regular-gitlab'
```

Comment:
We can skip the bitbucket and gitlab tests for the release CI because they have the same coverage for CS functionalities.

Reply:
Good catch - it wasn't part of our plan to include these. Removed